### PR TITLE
[UXP-3349] Detailed offer comparison for NGS

### PR DESCRIPTION
**Pull request by andrejak**
Adds slice fare card components to the table view to allow more detailed offer comparison:
Screenshot 2024-01-26 at 14 47 10
Screenshot 2024-01-26 at 14 47 22

### CODE REVIEW

File: src/components/DuffelNGSView/DuffelNGSView.tsx [hunk 1]
```diff
@@ -9,12 +9,81 @@ import {
...
+  switch (shelf) {
+    case "1":
+      return null;
+    case "2":
+      return "1";
+    case "3":
+      return "2";
+    case "4":
+      return "3";
+    case "5":
```
Comment (andrejak) [inline src/components/DuffelNGSView/DuffelNGSView.tsx:34]: Lmk if you think this is getting to weird given these are currently just numbers 1-5...

Comment (igorp1) [reply src/components/DuffelNGSView/DuffelNGSView.tsx:34]: mmm a bit weird I think, though pretty readable fwiw. How about 

```typescript
    function getPreviousShelf(shelf) {
       const previousShelf = +shelf - 1
       return previousShelf === 0 ? null : previousShelf.toString()
    }
```

**Review by igorp1 — APPROVED** [review]
Looks solid, thank you!

**Comment by srcltd** [comment]
Looking good! I hope there are no slices that only have basic or lux as it looks weird haha

**Pull request merged**